core/system/src/mpu/vmpu_freescale_k64_aips.c

```diff
@@ -117,6 +109,16 @@ int vmpu_aips_add(uint8_t box_id, void* start, uint32_t size, UvisorBoxAcl acl)
     return 1;
 }
 
+void vmpu_aips_common_add(void)
+{
+    int box, slot;
+    for (box = 1; box <= g_vmpu_box_count; box++) {
```

Comment:
`g_vmpu_box_count` can be `UVISOR_MAX_BOXES` - See: `if((box_id = g_vmpu_box_count++)>=UVISOR_MAX_BOXES)` in `/core/system/src/mpu/vmpu.c`

Should change condition to `box < g_vmpu_box_count`, so not to overflow `g_aipsx_box` buffer.

Alternatively change `if((box_id = g_vmpu_box_count++)>=UVISOR_MAX_BOXES)` to `if((box_id = ++g_vmpu_box_count)>=UVISOR_MAX_BOXES)`